frontend/packages/multicluster-sdk/src/api/index.ts

```diff
@@ -2,7 +2,7 @@
 export * from './constants'
 export * from './FleetResourceLink'
 export * from './useFleetK8sAPIPath'
-export * from './useFleetK8sWatchResource'
+export { useFleetK8sWatchResource } from './use-fleet-k8s-watch-resource/use-fleet-k8s-watch-resource'
```

Comment:
This is the internal version of the hook. We don't want this exported. I would actually like these files moved under `src/internal` ideally. (The original line was correct.)